microsoft-edge/devtools-guide-chromium/whats-new/2023/04/devtools-112.md

```diff
@@ -0,0 +1,176 @@
+---
+title: What's new in DevTools (Microsoft Edge 112)
+description: "The Performance tool can now unminify JavaScript running in out-of-process iframes, the CSS Overview tool displays a list of non-simple selectors for quick performance wins, and more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 04/08/2023
+---
+# What's New in DevTools (Microsoft Edge 112)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Performance tool unminifies JavaScript running in out-of-process iframes
+<!-- ## The Performance tool can now unminify JavaScript running in out-of-process iframes -->
+
+<!-- Subtitle: As long as DevTools can load your sourcemaps, you now have unparalleled insight into JavaScript execution within an iframe. -->
+
+In previous versions of Microsoft Edge, JavaScript execution in an out-of-process iframe (OOPIF) was isolated from DevTools.  As a result, the **Performance** tool was not able to apply sourcemaps to performance events that are from an OOPIF.  When recording a profile in the **Performance** tool, the tool can only display the unminified file and function names in the flame chart:
+
+![The minified call stack from a profile recorded in the Performance tool](./devtools-112-images/unminification-in-oopifs-before.png)
+<!--  unminified? -->
+
+In Microsoft Edge 112, the **Performance** tool now loads and applies sourcemaps to the performance events that are tracked in OOPIFs.  As a result, after recording a profile in the **Performance** tool, you can now see unminified file and function names for the JavaScript execution that occurs within an OOPIF:
+
+![The unminified call stack from a profile recorded in the Performance tool](./devtools-112-images/unminification-in-oopifs-after.png)
+
+See also:
+* [Performance tool unminifies file and function names when profiling production code](./devtools-111.md#performance-tool-unminifies-file-and-function-names-when-profiling-production-code) in _What's new in DevTools (Microsoft Edge 111)_.
+* [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md)
+* [Securely debug original code by using Azure Artifacts symbol server source maps](../../../javascript/consume-source-maps-from-azure.md)
+* [Record performance](../../../evaluate-performance/reference.md#record-performance) in _Performance features reference_.
+
+
+<!-- ====================================================================== -->
+## CSS Overview tool displays non-simple selectors for quick performance gains
+<!-- ## The CSS Overview tool displays a list of non-simple selectors for quick performance wins -->
```

Comment:
2.  @zoherghadyali - use this new, telegraphic (shortened) heading, or the original heading that's hidden in a comment?